R/pivot-long.R

```diff
@@ -140,6 +140,7 @@
 #'   )
 pivot_longer <- function(data,
                          cols,
+                         ...,
```

Comment:
In the `pivot_longer/wider()` docs these dots are documented as "passed on to methods"

But in the `build_*()` functions they must be empty